### PR TITLE
Add service layer

### PR DESCRIPTION
**Pull request by ponyataeva**

### CODE REVIEW

File: core/src/main/java/com/tweet/core/repository/TweetRepository.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.tweet.core.repository;
+
+import com.tweet.core.model.Tweet;
+import org.springframework.data.jpa.repository.JpaRepository;
+
```
Comment (ManWithNoNamePL) [inline core/src/main/java/com/tweet/core/repository/TweetRepository.java]: Please add @Repository annotation so spring can find this bean.
Beyond that, it activates persistence exception translation for all beans annotated with @Repository, to let exceptions being thrown by the JPA persistence providers be converted into Spring’s DataAccessException hierarchy.

File: core/src/main/java/com/tweet/core/repository/UserRepository.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.tweet.core.repository;
+
+import com.tweet.core.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
```
Comment (ManWithNoNamePL) [inline core/src/main/java/com/tweet/core/repository/UserRepository.java]: Same as in previous repo, please add @Repository annotation.

File: core/src/main/java/com/tweet/core/service/impl/UserServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package com.tweet.core.service.impl;
+
+import com.tweet.core.model.User;
+import com.tweet.core.repository.UserRepository;
+import com.tweet.core.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class UserServiceImpl implements UserService {
```
Comment (ManWithNoNamePL) [inline core/src/main/java/com/tweet/core/service/impl/UserServiceImpl.java]: @Service

**Comment by ponyataeva** [comment]
Fixed

**Pull request merged**